### ISSUE TITLE
AWS console login broken for a few users

### ISSUE THREAD

**Issue by davidread** [ministryofjustice/analytics-platform#70]
User clicks on "Open on AWS" button (control panel) and gets this:
![image 3](https://user-images.githubusercontent.com/307612/46023746-38ea8880-c0dd-11e8-8751-5b6ca405f9d9.png)

Occurs for OL, RMcC, JB

Whereas is works for us admins...

We've checked the computer clock is accurate.

## Steps to reproduce the problem

Consistent for these particular users.

**Comment by r-mccormack** [comment]
Just for clarity. I currently have this issue when attempting to access a bucket from my DOM1 machine, but it works fine when I try from my MacBook.

**Comment by davidread** [comment]
Work-around: https://alpha-analytics-moj.eu.auth0.com/samlp/NpfImg4P3ynU6HFx7ivYmqUZWQHfwi3Y

**Comment by davidread** [comment]
@r-mccormack can you confirm if this is still an issue?

**Comment by r-mccormack** [comment]
Hi @davidread , I just checked and it is. As before it works on my Mac, but fails with the message in the screenshot on my DOM1 machine.

**Comment by RobinL** [comment]
AW also has experienced this today

**Comment by MeganStodel** [comment]
This has started happening to me (yesterday PM, still the case this AM)

**Comment by jamiefraser77** [comment]
I got this error today as well

**Comment by djsd123** [comment]
@jamiefraser77 Did this occur while using your DOM1 machine?

**Comment by xoen** [comment]
Hello, I just got this error, I clicked on the logout button and tried again and it worked.

**Workaround**
1. Logout by following "click here" link in the AWS error page (**not** Control Panel, AWS)
2. Go back to Control Panel and click on "Open on AWS" button again

*Hopefully this workaround work for you as well.*

(I'm not 100% what's happening here but I suspect somehow this happens when I'm already logged in but with different credentials)
